src/fides/api/alembic/migrations/versions/3038667ba898_update_system_legal_basis_fields.py

```diff
@@ -0,0 +1,154 @@
+"""update system.legal_basis_for_transfers
+
+Revision ID: 3038667ba898
+Revises: 507563f6f8d4
+Create Date: 2023-08-25 14:19:16.798409
+
+"""
+from typing import Dict
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.orm.session import Session
+from sqlalchemy.sql import text
+
+# revision identifiers, used by Alembic.
+revision = "3038667ba898"
+down_revision = "507563f6f8d4"
+branch_labels = None
+depends_on = None
+
+legal_basis_transfers_upgrades: Dict[str, str] = {
+    "Adequacy decision": "Adequacy Decision",
+    "Standard contractual clauses": "SCCs",
+    "Binding corporate rules": "BCRs",
+}
+
+legal_basis_transfers_downgrades: Dict[str, str] = {
+    value: key for key, value in legal_basis_transfers_upgrades.items()
+}
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+    conn = session.connection()
+    bind = op.get_bind()
+
+    #############################
+    # legal_basis_for_transfers #
+    #############################
+    statement_transfers = text(
+        """SELECT distinct legal_basis_for_transfers, id
+FROM ctl_systems"""
+    )
+    existing_values_transfers = conn.execute(statement_transfers)
+
+    # drop column since we will repopulating later
+    op.drop_column(
+        "ctl_systems",
+        "legal_basis_for_transfers",
+    )
+
+    # add column back, but make it multi-value
+    op.add_column(
+        "ctl_systems",
+        sa.Column(
+            "legal_basis_for_transfers",
+            type_=sa.ARRAY(sa.String()),
+            server_default="{}",
+            nullable=True,
+        ),
+    )
+
+    # data migration - updates to some values since allowable values enum has changed
+    # (enum is not enforced/specified on the DB, just API-level)
+
+    for row in existing_values_transfers:
+        existing_value = row[0]
+        updated_value = legal_basis_transfers_upgrades.get(
+            existing_value, existing_value
+        )
+        updated_value = [
+            updated_value
+        ]  # wrap in a list because column is now array type
```

Comment:
Early review comment, apologies if I'm butting in too early - make sure updated_value is not null so you don't get {NULL}

Reply:
never too early, thank you!!! good catch 🙏

Reply:
addressed in [1857c7b](https://github.com/ethyca/fides/pull/3977/commits/1857c7b398969554f8d0e0192d674813c7ce0614) - does that look good to you? seemed to work for me locally on a manual smoke test.